modules/standard/Path.chpl

```diff
@@ -120,20 +120,27 @@ private inline proc unescape(str: string) {
     reliance on :proc:`~FileSystem.locale.cwd()`. Another task on the current
     locale may change the current working directory at any time.
 
-  :arg name: The path whose absolute path is desired.
-  :type name: `string`
+  :arg path: The path whose absolute path is desired.
+  :type path: `string`
 
   :return: A normalized, absolutized version of the path specified.
   :rtype: `string`
 
   :throws SystemError: Upon failure to get the current working directory.
 */
-proc absPath(name: string): string throws {
+proc absPath(path: string): string throws {
   use FileSystem;
 
-  if !isAbsPath(name) then
-    return normPath(joinPath(try here.cwd(), name));
-  return normPath(name);
+  if !isAbsPath(path) then
+    return normPath(joinPath(try here.cwd(), path));
+  return normPath(path);
+}
+
+pragma "no doc"
+pragma "last resort"
+proc absPath(name: string): string throws {
+  compilerWarning('Argument `name` is deprecated - use `path` instead');
```

Comment:
Can you prefix the deprecation message with the function name? Maybe even with `Path.` qualifier? So it'd be something like:

```
Path.absPath: Argument `name` is deprecated - use `path` instead
```

(Please do the same for others)

Reply:
In case this was something you were wondering: in Chapel you can have either single- or double- quoted strings. There's no major difference between the two, and you can use the other-kind-of-quote without escaping in them. So, you could also do this by either:

```chapel
compilerWarning('Argument "name" is deprecated - use "path" instead');
compilerWarning("Argument 'name' is deprecated - use 'path' instead");
```

Not to suggest you make any such change now :)